### ISSUE TITLE
youtube-subs-grid incompatible with Firefox Multi-Process.

### ISSUE THREAD

**Issue by Lebon14** [CodeNinjaUK/youtube-subs-grid#98]
As the title states, the addon is incompatible with Firefox' Electrolyte, ie. Multi-Process as detected by Mozilla's Addon Compatibility Reporter: https://addons.mozilla.org/en-US/firefox/addon/add-on-compatibility-reporter/

Could we have it fixed?
Thanks!

**Comment by DanSmith70** [comment]
Yep absolutely, with the WebExtensions API having been out for a while now my plan is to simplify the code a bunch and remove the need to have separate code for the FF and Chrome versions. This should be inherently compatible with Electrolyte as well. Hoping I can work on that this week!
